structure.md

```diff
@@ -0,0 +1,29 @@
+# Structure
+
+Having a good structure is essential for our culture. We derive and maintain our structure from the learnings of the open-source community to maximize the odds to [better decisions](decision-making.md), innovation, and sustainability.
+
+Presently, our structure constitutes of the following supporting natures.
```

Comment:
Natures is a bit ambiguous perhaps use {points, items, ideas} or {environments, philosophies}